experimental/packages/exporter-trace-otlp-grpc/package.json

```diff
@@ -65,13 +65,13 @@
     "typescript": "4.4.4"
   },
   "peerDependencies": {
-    "@opentelemetry/api": "^1.0.3"
+    "@opentelemetry/api": ">=1.0.0 <1.2.0"
   },
   "dependencies": {
     "@grpc/grpc-js": "^1.3.7",
```

Comment:
You should move `@grpc/grpc-js` to the same version (e.g. `^1.5.9`) to solved the current build problems